### ISSUE TITLE
BUG - Invalid URL error when entering a valid PR link during onboarding 

### ISSUE THREAD

**Issue by ladonnaq** [Azure/azure-sdk-tools#7165]
There seems to be a bug during onboarding. 

Service Name: Fairfield Gardens
Offering Name: Fairfield Gardens 

PR Links: 
Data Plane PR: https://github.com/Azure/azure-rest-api-specs-pr/issues/15060
Management Plane PR: https://github.com/Azure/azure-rest-api-specs-pr/pull/15107

![image](https://github.com/Azure/azure-sdk-tools/assets/67656762/729546dd-c27b-4413-a651-e6f051e17d67)

**Comment by maririos** [comment]
@ladonnaq 

> Data Plane PR: https://github.com/Azure/azure-rest-api-specs-pr/issues/15060

This is incorrect. They are linking a GH issue that tracks the stewardship meeting. See how the URL has `issues` in it instead or `pull`. Looking at that issue, the PR for data plane is https://github.com/Azure/azure-rest-api-specs-pr/pull/15290
